base_max_linear_x_vel: 1.0          # m per second in run mode
base_max_linear_y_vel: 1.0          # m per second in run mode
base_max_angular_vel: 1.5           # rad per second in run mode
soft_joint_limit_threshold: 0.1     # rad
